src/plugins/workspace/common/constants.ts

```diff
@@ -16,3 +18,7 @@ export enum WorkspacePermissionMode {
   LibraryRead = 'library_read',
   LibraryWrite = 'library_write',
 }
+
+export const PATHS = {
```

Comment:
if `src/plugins/workspace/public/components/route.ts` is not used, then we don't need this variable as well.